### PR TITLE
[ci] Added a spec for ProjectController#status.

### PR DESCRIPTION
**Pull request by evanrolfe**
https://trello.com/c/t4lBVvEw/511-3-p7-write-test-for-app-controllers-webui-projectcontrollerrbstatus

### CODE REVIEW

**Comment by evanrolfe** [comment]
[(3) P7: Write test for app/controllers/webui/project_controller.rb#status](https://trello.com/c/t4lBVvEw/511-3-p7-write-test-for-app-controllers-webui-projectcontrollerrbstatus)

File: src/api/spec/controllers/webui/project_controller_spec.rb [hunk 1]
```diff
@@ -1658,5 +1658,61 @@
...
+      get :status, params: params
+    end
+
+    context 'no params set' do
+      let(:params) { { project: apache_project.name } }
+
+      it_behaves_like 'a project status controller'
+    end
+
+    context 'no param format=json set' do
```
Comment (Ana06) [inline src/api/spec/controllers/webui/project_controller_spec.rb]: shouldn't this be `context 'param format=json set' do`? :confused:

Comment (evanrolfe) [inline src/api/spec/controllers/webui/project_controller_spec.rb]: ah yes.. fixed.

**Comment by evanrolfe** [comment]
@mschnitzer git-cop requires that the commit message start with a word like "Fixed", "Added" etc. so if you want it to present tense then you should open a PR to change the git-cop config.

@bgeuken all the cassettes are needed. They are all located in this dir which belongs to the test I've added: `spec/cassettes/Webui_ProjectController/GET_status/`

@Ana06 please re-review

**Comment by evanrolfe** [comment]
@Ana06 ping

**Comment by Ana06** [comment]
@evanrolfe 

> @mschnitzer git-cop requires that the commit message start with a word like "Fixed", "Added" etc. so if you want it to present tense then you should open a PR to change the git-cop config.

That is git-cop default configuration, not our. So, @mschnitzer is right we should better use present. I will implement a rake task to run git-cop locally and add some documentation in this regard too so anybody else has the same misunderstanding. :wink:

**Comment by evanrolfe** [comment]
The Codecov check seems to be stuck but travis and hakiri are both passing. I will merge this tomorrow unless anybody has objections?

**Comment by bgeuken** [comment]
@evanrolfe @openSUSE/open-build-service You will need to rebase to a newer master state. It's explained in their doc (just click the documentation link [here](https://codecov.io/gh/openSUSE/open-build-service/pull/3652).

Here is the short version:
```
In order to produce an accurate coverage comparison, Codecov must have the coverage report for the pull request base.

...

This issue occurs when the base commit (the parent commit of the first commit on the pull request) did not upload coverage and/or failed CI tests.
```

**Comment by evanrolfe** [comment]
> You will need to rebase to a newer master state. It's explained in their doc (just click the documentation link [here](https://codecov.io/gh/openSUSE/open-build-service/pull/3652).

@bgeuken thanks for checking that for me.

**Pull request merged**